dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryInodesForm.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.rest.api.v1.categories;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.List;
+
+/**
+ * Category Input Form
+ *
+ * @author Hassan Mustafa Baig
```

Comment:
fix this

Reply:
done https://github.com/dotCMS/core/pull/29142/commits/7084e896746daa8a435e4850fe8504190bfd78bb#diff-5ed5ece5f60f8df0f26394221cde26c80b43e0541f6b789ad0072f12c88c2cffL13